processing/data_collection/gazette/spiders/ap_macapa.py

```diff
@@ -0,0 +1,73 @@
+import datetime as dt
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class ApMacapaSpider(BaseGazetteSpider):
+
+    name = "ap_macapa"
+    allowed_domains = ["macapa.ap.gov.br"]
+    start_date = None
+
+    TERRITORY_ID = "1600303"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        self.start_date = dt.date.today().replace(day=1)
```

Comment:
Please, set the `start_date` to the first date with gazettes available. Other spiders already do that as well. Thanks

Reply:
I though the idea was having incremental crawls, but ok, I'll do it.